packages/cfd/src/Components/props.types.ts

```diff
@@ -68,6 +68,53 @@ export type TTradingPlatformAvailableAccount = {
     sub_account_type: string;
 };
 
+export type TCardFlipStatus = {
+    svg: boolean;
+    bvi: boolean;
+    labuan: boolean;
+    vanuatu: boolean;
+    maltainvest: boolean;
+};
+
+export type TClickableDescription = {
+    type: 'text' | 'link';
+    text: string;
+    linkData?: string;
+};
+
+export type TJurisdictionCardSectionTitleIndicators = {
+    type: 'displayText' | 'displayIcons';
+    displayText?: string;
+    displayTextSkinColor?: string;
+    displayIcons?: Array<string>;
+};
+
+export type TJurisdictionCardSection = {
+    key: string;
+    title: string;
+    titleIndicators?: TJurisdictionCardSectionTitleIndicators;
+    description?: string;
+    clickableDescription?: Array<TClickableDescription>;
+};
+
+export type TJurisdictionCardBackRequiredDocsItem = {
+    icon: string;
+    text: string;
+};
+
+export type TJurisdictionCardBackSectionRequiredDocs = {
+    documentNumber?: TJurisdictionCardBackRequiredDocsItem;
+    selfie?: TJurisdictionCardBackRequiredDocsItem;
+    identityDocument?: TJurisdictionCardBackRequiredDocsItem;
+    nameAndAddress?: TJurisdictionCardBackRequiredDocsItem;
```

Comment:
variable names are not following our naming conventions.
Variable names should be snake_case.
Function names are camelCase.